sources/PTYSession.m

```diff
@@ -205,6 +205,8 @@ @implementation PTYSession {
     VT100Terminal *_terminal;
 
     NSString *_termVariable;
+    
+    NSString *_answerBackString;
```

Comment:
Instead of doing
````
[self setAnswerBackString:[iTermProfilePreferences stringForKey:KEY_ANSWERBACK_STRING inProfile:aDict]];
````
do:
````
[_terminal setAnswerBackString:[iTermProfilePreferences stringForKey:KEY_ANSWERBACK_STRING inProfile:aDict]];
````

Then you can remove _answerBackString from this file.